docs/user/unattended/index.md

```diff
@@ -110,3 +110,12 @@ sudo agama config show > profile.json
 To start an unattended installation process, you need to tell Agama where to find the profile. When
 using the Live ISO, you must use the `inst.auto` boot option. Please, check the [boot
 options](../boot_options/index.md) for further information.
+
+If you do not specify any profile, Agama will automatically search for it in a few predefined
+locations. Agama expects a file named `autoinst.xml`, `autoinst.jsonnet` or `autoinst.json` (in that
+particular order) to be located on:
+
+- A file system named `OEMDRV`.
+- Or the root (`/`) of the installation environment.
+
+The first found file is used as the profile, starting the installation right away.
```

Comment:
i'm not an english guru ... but i somehow feel that "The first file found is used ..." sounds better.

May be @mvidner should do quick reading.